src/language/worker.ts

```diff
@@ -140,9 +140,9 @@ const getListDatabases = async (
     const result = await serviceProvider.listDatabases('admin');
     const databases = result
       ? result.databases.map((item) => ({
-          label: item.name,
-          kind: CompletionItemKind.Value
-        }))
+        label: item.name,
+        kind: CompletionItemKind.Value
+      }))
```

Comment:
Will this endless adding/deleting spaces be fixed by this PR with lint improvements?

Reply:
With https://github.com/mongodb-js/vscode/pull/231 yes